docs/articles/toc.yml

```diff
@@ -64,6 +64,8 @@
     href: persistence/storage-plugins.md
   - name: Custom serialization
     href: persistence/custom-serialization.md
+  - name: Writing A Custom Persistent Provider
+    href: persistence/custom-persistence-provider.md
   - name: Persistence Query
     href: persistence/persistence-query.md
   - name: Persistence Testing
```

Comment:
While we are here, can we create `toc.yml` for the persistence section and use that here? For example: `https://github.com/Arkatufus/akka.net/tree/master/docs/articles/intro/getting-started/toc.yml`

Reply:
That would be out of scope of this PR, you're mixing 2 kinds of issues